<launch>
    
    <node pkg="mouse_monitor" type="MouseMonitorBeagle" name="MouseMonitorBeagle" output="screen" />

</launch>
